`nepho cloudlet update` is broken

```
$ nepho cloudlet update nepho-example
Updating cloudlet: /Users/huit/.nepho/cloudlets/nepho-example
Traceback (most recent call last):
  File "/Users/huit/.virtualenvs/nepho/bin/nepho", line 8, in <module>
    execfile(__file__)
  File "/Users/huit/python/nepho/nepho/bin/nepho", line 18, in <module>
    sys.exit(main())
  File "/Users/huit/python/nepho/nepho/bin/nepho", line 13, in main
    app.run()
  File "/Users/huit/.virtualenvs/nepho/lib/python2.7/site-packages/cement-2.1.4.dev20131029203905-py2.7.egg/cement/core/foundation.py", line 493, in run
    self.controller._dispatch()
  File "/Users/huit/.virtualenvs/nepho/lib/python2.7/site-packages/cement-2.1.4.dev20131029203905-py2.7.egg/cement/core/controller.py", line 448, in _dispatch
    func()
  File "/Users/huit/.virtualenvs/nepho/lib/python2.7/site-packages/cement-2.1.4.dev20131029203905-py2.7.egg/cement/core/controller.py", line 454, in _dispatch
    func()
  File "/Users/huit/python/nepho/nepho/nepho/cli/cloudlet.py", line 179, in update
    c.update()
  File "/Users/huit/python/nepho/nepho/nepho/core/cloudlet.py", line 100, in update
    repo.remotes.origin.pull()
  File "/Users/huit/.virtualenvs/nepho/lib/python2.7/site-packages/GitPython-0.3.2.RC1-py2.7.egg/git/remote.py", line 605, in pull
    return self._get_fetch_info_from_stderr(proc, progress or RemoteProgress())
  File "/Users/huit/.virtualenvs/nepho/lib/python2.7/site-packages/GitPython-0.3.2.RC1-py2.7.egg/git/remote.py", line 539, in _get_fetch_info_from_stderr
    assert len(fetch_info_lines) == len(fetch_head_info), "len(%s) != len(%s)" % (fetch_head_info, fetch_info_lines)
AssertionError: len(["28160328b3b43f85508d7e650be5f037a3be68a6\t\tbranch 'master' of https://github.com/cloudlets/nepho-example\n", "aa0d418ff0951c5e6e2cb62cc0512c50f2ddf8f0\tnot-for-merge\tbranch 'bugfix/9-missing_log_timestamp' of https://github.com/cloudlets/nepho-example\n", "3433ec97a6ff5027aff7b07d9ab18164bea025a1\tnot-for-merge\tbranch 'feature/development-aws' of https://github.com/cloudlets/nepho-example\n", "a2cad770d08b5f546b0bed20ad6c781f504eea27\tnot-for-merge\ttag 'v1.0.0' of https://github.com/cloudlets/nepho-example\n"]) != len(['POST git-upload-pack (466 bytes)', '   ffd72de..2816032  master     -> origin/master', ' + e39f15a...aa0d418 bugfix/9-missing_log_timestamp -> origin/bugfix/9-missing_log_timestamp  (forced update)', ' = [up to date]      feature/development-aws -> origin/feature/development-aws', ' * [new tag]         v1.0.0     -> v1.0.0'])
```

so that this doesn't block the release, i'm going to trap the AssertionError to hide the unsightly traceback, and then we'll fix it in 1.1.0.

--- a/nepho/cli/cloudlet.py
+++ b/nepho/cli/cloudlet.py
@@ -176,7 +176,10 @@ class NephoCloudletController(base.NephoBaseController):
         if not isinstance(cl, list):
             cl = [cl]
         for c in cl:
-            c.update()
+            try:
+                c.update()
+            except AssertionError:
+                print colored("Error: ", "red") + "Cloudlet update failed, see issue #176."
 
     @controller.expose(help="Uninstall a Nepho cloudlet", aliases=["remove"])
     def uninstall(self):

